### PR TITLE
Support testing on multiple targets

### PR DESCRIPTION
**Pull request by HakkyuKim**
This PR supports running integration tests on multiple targets by passing a yaml file.

Given the following `recipe.yaml` file,
```
plugins:
  audioplayers: ["wearable-5.5"]
  package_info_plus: ["wearable-5.5", "mobile-6.0"]
  wakelock: ["tv-6.0"]
```

running `tools/run_command.py test --plugins wakelock audioplayers package_info_plus --recipe recipe.yaml` outputs the following result:

```
SUCCEEDED: wakelock tv-6.0
SUCCEEDED: audioplayers wearable-5.5
SUCCEEDED: package_info_plus wearable-5.5
SUCCEEDED: package_info_plus mobile-6.0
All tests passed!
```

## TODOs
- [x] handle error cases
- [x] formatting

### CODE REVIEW

File: tools/commands/integration_test.py [hunk 1]
```diff
@@ -103,95 +147,192 @@ def _integration_test(plugin_dir, timeout):
...
+                                               shell=True,
+                                               cwd='.',
+                                               universal_newlines=True,
+                                               stderr=subprocess.PIPE,
+                                               stdout=subprocess.PIPE)
+            profile, platform_version = _parse_target_info(
+                completed_process.stdout)
+            if not profile or not platform_version:
+                print(f'''
+Cannot extract {profile} or {platform_version} information from device {id}
```
Comment (bbrto21) [inline tools/commands/integration_test.py:273]: Maybe you meant this.

```suggestion
Cannot extract profile or platform_version information from device {id}
```

**Review by bbrto21 — COMMENTED** [review]
remove this `import argparse `

File: tools/commands/integration_test.py (hunk 1 above)
Comment (HakkyuKim) [reply tools/commands/integration_test.py]: I totally did, thanks!

**Review by bbrto21 — APPROVED** [review]
LGTM

**Comment by HakkyuKim** [comment]
> remove this `import argparse `

Oh, I missed this comment, I removed the import. I also noticed that you can't properly import `commands.command_utils as command_utils` when running scripts in `commands`. If you're not planning to run scripts independently, I'll remove all `__name__ == "__main__"` from command scripts.

**Comment by bbrto21** [comment]
> If you're not planning to run scripts independently, I'll remove all __name__ == "__main__" from command scripts.

Well.. I had have a plan not detailed... please feel free to remove them if you want :laughing:

**Pull request merged**